Mathlib/Data/Finset/Basic.lean

```diff
@@ -807,6 +807,11 @@ theorem ssubset_iff_exists_cons_subset : s ⊂ t ↔ ∃ (a : _) (h : a ∉ s),
   obtain ⟨a, hs, ht⟩ := not_subset.1 h.2
   exact ⟨a, ht, cons_subset.2 ⟨hs, h.subset⟩⟩
 
+theorem cons_swap (ha : a ∉ s) (hb : b ∉ s.cons a ha) :
```

Comment:
```suggestion
theorem cons_comm (ha : a ∉ s) (hb : b ∉ s.cons a ha) :
```
to match `Set.insert_comm`.

Could you add `Finset.insert_comm` as well?

Reply:
Does that imply `Multiset.cons_swap` should be also renamed to `Multiset.cons_comm`?

Reply:
Ouch, nice catch. Summoning @YaelDillies

Reply:
Also we already have [`insert_comm`](https://github.com/leanprover-community/mathlib4/blob/bc87839ae48935a0ee4540e49e93e32ac671d5b7/Mathlib/Data/Finset/Basic.lean#L993C1-L994C55).

Reply:
`Insert.comm` is surely misnamed as well! Can you create a zulip thread about this? I think we can merge your PR without addressing it, but it would be nice to clean up in a follow up.

Reply:
https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/Renaming.20theorems.20related.20to.20sets/near/475102498

Reply:
Thanks for [the thread](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/Renaming.20theorems.20related.20to.20sets/near/475102498)!

Reply:
Yes, this should be `cons_comm`, the multiset lemma should also renamed (and `forall_swap` too, while we are at it), and `insert_comm` already exists under the name `Insert.comm`...